PhysicsTools/IsolationAlgos/plugins/CITKPFIsolationSumProducerForPUPPI.cc

```diff
@@ -76,7 +76,10 @@ namespace citk {
         puppiValueMapToken_ = mayConsume<edm::ValueMap<float>>(c.getParameter<edm::InputTag>("puppiValueMap")); //getting token for puppiValueMap
         useValueMapForPUPPI = true;
       }
-      else useValueMapForPUPPI = false;
+      else {
+	useValueMapForPUPPI = false;
+	usePUPPINoLepton = (c.exists("usePUPPINoLepton") ? c.getParameter<bool>("usePUPPINoLepton") : false);
```

Comment:
fillDescriptions already defines the usePUPPINoLepton parameter and its default value (false).
Please change this to ```usePUPPINoLepton = c.getParameter<bool>("usePUPPINoLepton")```